docs/home/howto/BACKUP.md

```diff
@@ -0,0 +1,175 @@
+## Epiphany backup and restore
+
+### Introduction
+
+Epiphany provides solution to create full or partial backup and restore for some components, like:
+
+- [Load Balancer](#load-balancer)
+- [Logging](#logging)
+- [Monitoring](#monitoring)
+- [Postgresql](#postgresql)
+- [RabbitMQ](#rabbitmq)
+- [Kubernetes (only backup)](#kubernetes)
+
+Backup is created directly on machine where component is running, and is moved to ``repository`` host rsync. On ``repository`` host backup files are stored in location ``/epibackup/mounted`` on local. 
+See [How to store backup](#2.-How-to-store-backup) chapter.
+
+## 1. How to perform backup
+
+#### Run one-line epicli:
+
+Run epicli backup with option ``-c`` passing component to back up :
+```
+epicli backup -f cluster-config.yml -b build_folder -c list,of,components
+```
+where ``-c`` might be list one or more comma separated components:
+```
+- c load_balancer,logging,monitoring,postgresql,rabbitmq
+```
+
+#### Create static configuration
+
+Enalbe backup for chosen components by setting up ``enabled`` parameter according your needs. Copy default configuration for backup from ``defaults/configuration/backup.yml`` into basic data yaml and modify ``enabled`` value.
+
+```
+kind: configuration/backup
+title: Backup Config
+name: default
+specification:
+  components:
+    load_balancer:
+      enabled: true
+    logging:
+      enabled: false
+    monitoring:
+      enabled: true
+    postgresql:
+      enabled: true
+    rabbitmq:
+      enabled: false
+#Kubernes backup and recovery is not supported by Epiphany.
+#You may create backup by enabling this below, but recovery should be done manually according Kubernetes documentation.
+    kubernetes:
+      enabled: false
+```
+
+Run ``epicli backup`` command:
+
+``epicli backup -f cluster-config.yml -b build_folder``
+
+In this case no need to pass parameter ``-c`` but if provided this will overwrite state from cluster-config.yml
+
+## 2. How to store backup
+
+Backup location is defined in ``backup`` role as ``backup_destination_host`` and ``backup_destination_dir``.
+Default backup location is defined on ``repository`` host in location ``/epibackup/mounted/``.
+Use ``mounted`` location as mount point and mount storage you want to use. This might be:
+- Azure Blob Storage
+- Amazon S3
+- NAS
+- Any other attached storage
+
+Ensure that mounted location has enought space, is reliable and is well protected against disaster.
```

Comment:
Ensure that the mounted location has **enough** space, is reliable and is well protected against disaster.